Don't force build-dir-in-build-dir logic, symlink the os-specific SCSI
support file in place.

Status: not suitable for upstream.

Signed-off-by: Gustavo Zacarias <gustavo.zacarias@free-electrons.com>

diff -Nura Makefile.linux Makefile.linux
--- a/Makefile.linux	2016-01-26 09:10:03.939963780 -0300
+++ b/Makefile.linux	2016-01-26 09:10:27.140763863 -0300
@@ -129,7 +129,7 @@
 	    echo "Please specify OS={aix,linux,hpux,solaris,windows}"; \
 	    exit 1; \
 	fi; \
-	ln -sf ../scsilib-$(OS).c scsilib.c
+	ln -sf scsilib-$(OS).c scsilib.c
 
 print:;
 		@$(PRINTER) $(PRINTFLAGS) $(ALL_CFILES)
